contrib/cirrus/lib.sh

```diff
@@ -183,30 +178,16 @@ setup_rootless() {
     cat $HOME/.ssh/*.pub /home/$ROOTLESS_USER/.ssh/*.pub >> $HOME/.ssh/authorized_keys
     cat $HOME/.ssh/*.pub /home/$ROOTLESS_USER/.ssh/*.pub >> /home/$ROOTLESS_USER/.ssh/authorized_keys
 
-    msg "Ensure the ssh daemon is up and running within 5 minutes"
-    systemctl start sshd
-    lilto systemctl is-active sshd
-
     msg "Configure ssh file permissions"
     chmod -R 700 "$HOME/.ssh"
     chmod -R 700 "/home/$ROOTLESS_USER/.ssh"
     chown -R $ROOTLESS_USER:$ROOTLESS_USER "/home/$ROOTLESS_USER/.ssh"
 
     msg "   setup known_hosts for $USER"
-    ssh -q root@localhost \
-        -o UserKnownHostsFile=/root/.ssh/known_hosts \
-        -o UpdateHostKeys=yes \
-        -o StrictHostKeyChecking=no \
-        -o CheckHostIP=no \
-        true
-
+    ssh-keyscan localhost > /root/.ssh/known_hosts
```

Comment:
Slight preference for `>>` (append instead of clobber). I suspect there's nothing valuable in that file (if it even exists), but it seems better in principle.

Also, will any of this code ever run on a system with SELinux enabled? If so you may want to `chmod 600` and `restorecon` the file. Likewise with user file below.

Reply:
There should never be a case where anything is trying to ssh-out from the test VM to anything other than localhost (it would indicate a security problem AFAIC).  We *want* the absolute authority over the contents in this case, so overwriting was intentional.  I can add a comment to that effect if you think it would help.